docs/serving/samples/autoscale-go/README.md

```diff
@@ -118,7 +118,7 @@ A demonstration of the autoscaling capabilities of a Knative Serving Revision.
 
 Knative Serving autoscaling is based on the average number of in-flight requests
 per pod (concurrency). The system has a default
-[target concurrency of 100](https://github.com/knative/serving/blob/master/config/core/configmaps/autoscaler.yaml#L54-L60)
+[target concurrency of 70](https://github.com/knative/serving/blob/8c7134ebfe368ee178229e99166e5bff8167b8ed/config/core/configmaps/autoscaler.yaml#L40-L55)
```

Comment:
No target concurrency is 100 by default.
70 is the hotness of revision
When multiplied we get autoscaling target of 70.